ci/dev/test-scripts.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+main() {
+  cd "$(dirname "$0")/../.."
```

Comment:
If you're trying to identify the root of the repository, this might help:

`PROJECT_ROOT="$(git rev-parse --show-toplevel)"`

Then you can `cd $PROJECT_ROOT`, though I'd suggest doing pushd/popd like this:

```
pushd $PROJECT_ROOT
  bats ./test/scripts
popd
```

You can also redirect to /dev/null if you want to suppress the default messages that pushd/popd will emit

Reply:
True! I tend to favor `dirname "$0"` because I can run it without having to `cd` to the repository first. It feels a little more flexible overall to me (at the cost of having to update the `cd` if the file is moved of course).